policy-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/policy/PolicyTestCommon.java

```diff
@@ -15,77 +15,65 @@
  */
 package uk.gov.gchq.palisade.contract.policy;
 
-import uk.gov.gchq.palisade.Context;
-import uk.gov.gchq.palisade.User;
 import uk.gov.gchq.palisade.policy.IsTextResourceRule;
 import uk.gov.gchq.palisade.policy.PassThroughRule;
 import uk.gov.gchq.palisade.resource.LeafResource;
-import uk.gov.gchq.palisade.resource.Resource;
 import uk.gov.gchq.palisade.resource.impl.DirectoryResource;
 import uk.gov.gchq.palisade.resource.impl.FileResource;
 import uk.gov.gchq.palisade.resource.impl.SystemResource;
 import uk.gov.gchq.palisade.rule.PredicateRule;
-import uk.gov.gchq.palisade.service.request.Policy;
+import uk.gov.gchq.palisade.rule.Rules;
 
 import java.util.Arrays;
 import java.util.HashSet;
 import java.util.Set;
 
-public class PolicyTestData {
-    public static final User USER = new User().userId("testUser");
-    public static final User SECRET_USER = new User().userId("secretTestUser").addAuths(new HashSet<>(Arrays.asList("Sensitive", "Secret")));
-    public static final Context CONTEXT = new Context().purpose("Testing");
-
+public class PolicyTestCommon {
     /**
      * Setup a collection of resources with policies like so:
      * /txt - only txt type files are viewable
-     *   /txt/json - only json format files are viewable
-     *     /txt/json/json.txt - an accessible json txt file
-     *     /txt/json/json.avro - an inaccessible json avro file (breaks /txt rule)
-     *     /txt/json/pickled.txt - an inaccessible pickle txt file (breaks /txt/json rule)
-     *   /txt/sensitive - only users with sensitive auth can view
-     *     /txt/sensitive/report.txt - an accessible (to sensitive auths) txt file
-     *     /txt/sensitive/salary.csv - an inaccessible csv file (breaks /txt rule)
-     *   /txt/secret - only users with secret auth can view, a purpose of testing will redact all record-level info
-     *     /txt/secret/secrets.txt - an accessible (to secret auths) txt file
+     * /txt/json - only json format files are viewable
+     * /txt/json/json.txt - an accessible json txt file
+     * /txt/json/json.avro - an inaccessible json avro file (breaks /txt rule)
+     * /txt/json/pickled.txt - an inaccessible pickle txt file (breaks /txt/json rule)
+     * /txt/sensitive - only users with sensitive auth can view
+     * /txt/sensitive/report.txt - an accessible (to sensitive auths) txt file
+     * /txt/sensitive/salary.csv - an inaccessible csv file (breaks /txt rule)
+     * /txt/secret - only users with secret auth can view, a purpose of testing will redact all record-level info
+     * /txt/secret/secrets.txt - an accessible (to secret auths) txt file
```

Comment:
I'd prefer to keep the indentation for clarity.
If autoformat keeps changing this, replace the spaces with `.`s

Reply:
added '.''s to prevent autoformatting messing it up again